requirements.txt

```diff
@@ -2,6 +2,7 @@ aiohttp
 altair>=4.2.0
 fastapi
 ffmpy
+git+https://github.com/huggingface/huggingface_hub.git
```

Comment:
will replace with`huggingface_hub` when `send_telemetry` is released.

Reply:
In `flagging.py`, we check to see if `huggingface_hub` is installed. We can now remove that check

Reply:
Also WDYT about pinning to a specific commit hash instead of installing the latest on `main` so that we have a stable, reliable install. Otherwise if anything breaks on `huggingface_hub`'s `main`, it could affect `gradio` users

Reply:
We can pin to the version of the next release of `huggingface_hub`, instead of a specific commit hash right?

Reply:
Once it's out on `huggingface_hub`, then I don't think we need to pin anymore, we can trust that `huggingface_hub` won't break on release. However, maybe I'm being paranoid, but I don't think we should assume that for `main`. Especially since our analytics happen early on, so if something breaks with analytics, it might prevent usage of gradio altogether